dietpi/dietpi-software

```diff
@@ -12236,7 +12253,33 @@ _EOF_
 			fi
 
 		fi
+		
+		software_id=185 # Portainer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
 
+			# Restart Docker service to be able to deploy the container
+			G_DIETPI-NOTIFY 2 'Docker will be restarted to be able to deploy the container.'
+			G_EXEC systemctl daemon-reload
+			G_EXEC systemctl restart docker.service
+			
+			# Check for existing portainer installation
+			if [[ -d '/mnt/dietpi_userdata/docker-data/volumes/portainer_data' ]]; then
+				G_DIETPI-NOTIFY 2 'Portainer detected, container will be dropped before reinstallation.'
+				G_EXEC docker rm "$(docker ps -a | grep "portainer/portainer-ce" | awk '{print $1}')" --force
+				G_EXEC docker rmi "$(docker images -a | grep "portainer/portainer-ce" | awk '{print $3}')"
+			else
+				# Create data directory
+				G_EXEC docker volume create portainer_data				
+			fi
+
+			# Deploy the portainer container
+			G_DIETPI-NOTIFY 2 'Portainer will be deployed now. This could take a while...'
+			G_EXEC docker run -d -p 9000:9000 --name=portainer --restart=always -v /var/run/docker.sock:/var/run/docker.sock -v portainer_data:/data portainer/portainer-ce
```

Comment:
I guess this contains some interesting output? In case, it can be shown on console via:
```
G_EXEC_OUTPUT=1 G_EXEC docker run ...
```

Reply:
ok this will show download and extraction of files

Reply:
Especially if this takes a bid longer it might be good to know the current step 👍.

Reply:
execution depends on device. On a RPi3 or 4 it's quite fast but on RPiB, will it take some minutes